packages/dashboard/src/components/grid/gestures/usePointerTracker.ts

```diff
@@ -39,9 +39,12 @@ export const usePointerTracker = ({ readOnly, enabled, union, click }: PointerTr
   };
 
   const onPointerUp: PointerEventHandler = (e) => {
+    const target = e.target as HTMLDivElement;
+
     if (cancelClick || !enabled || readOnly) return;
 
-    if (e.button === MouseClick.Left) {
+    /* handle classname 'awsui_modal' to keep focus on selected widget when clicking on modal */
+    if (e.button === MouseClick.Left && !target.ownerDocument.body.className.includes('awsui_modal')) {
```

Comment:
should use a ref instead, we cannot depend on cloudscape classNames to select elements

Reply:
you can instead disable the grid when the modal is show so that none of the gestures are dispatched if the grid is not the focus.

Reply:
There is an action for this - `packages/dashboard/src/store/actions/changeDashboardGrid/changeEnabled.ts`

Reply:
Disabled grid when the modal is opened